cylc/uiserver/workflows_mgr.py

```diff
@@ -80,7 +91,7 @@
             if flags.debug:
                 raise
             logger.error("ERROR: %s: %s\n", exc, host)
-            return (reg, host, port, pub_port, None)
+            return (reg, host, port, pub_port, None, None)
```

Comment:
Further down, this function returns a tuple with 6 elements instead of 5. So callers of this API need to either check the elements of the tuple, then unpack or iterate it correctly. Now returning always the 6 elements users can ignore that element counting step.